src/Entity/CourseUser.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: grantas
+ * Date: 18.5.1
+ * Time: 14.37
+ */
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\CourseUserRepository")
+ */
+class CourseUser
+{
+
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $user;
```

Comment:
Nelabai suprantu dėlei kokių priežasčių čia yra integer'is... Negana to, jei jau id tiktai, tai name'ingas klaidinantis tuomet, nes natūraliai tikimasi userio, o ne jo id čia. Jums doctrine'a ir duota naudotis tam, kad galėtumėt relation'us dėliot. Čia turėtų būti `ManyToOne` relation'as ir `$user` turėtų būti userio instance'as, o ne id.